sass/main.scss

```diff
@@ -4,12 +4,39 @@
 @import "elements/elements";
 
 
+.ellipsis {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+html {
+  font-size: 12px;
+}
+
+@include media-breakpoint-up(sm) {
+  html {
+    font-size: 16px;
```

Comment:
This was just an example and I thought we could modify it to something more suited to different screen sizes, currently I only have my laptop to develop on so I hadn't specified different font sizes for the range of screens, it is possible that we only need the two (mobile 12px and desktop 16px)